src/simple-scrollbar.js

```diff
@@ -37,7 +37,36 @@
       d.removeEventListener('mouseup', stop);
     }
   }
-
+  function getHiddenParent(element) {
+    var hiddenParent = "";
+    while (element.parentNode !== null && element.parentNode.offsetHeight === 0) {
+      if (element.parentNode.style.display === "none") {
```

Comment:
this is incorrect. if u add a new test that applies a class to body to make it invisible, the test will fail
if u add another test that set scrollbar's viewport to display:none and then set viewport's container to display:none, it will also fail